toolchain/check/BUILD

```diff
@@ -72,6 +72,7 @@ cc_library(
         "//common:array_stack",
         "//common:check",
         "//common:map",
+        "//common:ostream",
```

Comment:
This is no longer used in this PR and can be removed.

Reply:
Removing